jetbrains-core/resources/META-INF/ext-rider.xml

```diff
@@ -10,6 +10,9 @@
         <component>
             <implementation-class>software.aws.toolkits.jetbrains.services.lambda.LambdaPsiHost</implementation-class>
         </component>
+        <component>
+            <implementation-class>software.aws.toolkits.jetbrains.settings.LambdaGutterMarkSettings</implementation-class>
```

Comment:
make it a service

Reply:
<projectService serviceImplementation="" /> 
inheritance from PersistentStateComponent is okay